From d1ecc836fd141b689339e2397066499ef5ec8afa Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Mon, 7 May 2018 14:17:05 +0200
Subject: [PATCH 0613/1051] arm64: dts: marvell: 8082-db: add missing comphy
 references

Adding phandle to particular nodes will allow relevant drivers to configure
appropriate marvell, comphy-cp110 accordingly.

Change-Id: I42b1d979765c17692e3bc3d127384619f03e7432
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8082-db.dts | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8082-db.dts b/arch/arm64/boot/dts/marvell/armada-8082-db.dts
index 0566fe767b04..2a1a1f828544 100644
--- a/arch/arm64/boot/dts/marvell/armada-8082-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8082-db.dts
@@ -74,12 +74,21 @@
 	sata-port@1 {
 		reg = <1>;
 		status = "okay";
+		/* Generic PHY, providing serdes lanes */
+		phys = <&cp0_comphy5 1>;
 	};
 };
 
 &cp0_pcie0 {
 	status = "okay";
 	msi-parent =<&ap0_gic_its>;
+
+	/* Generic PHY, providing serdes lanes */
+	phys = <&cp0_comphy0 0
+		&cp0_comphy1 0
+		&cp0_comphy2 0
+		&cp0_comphy3 0>;
+	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp0_pcie1 {
@@ -95,12 +104,21 @@
 	sata-port@1 {
 		reg = <1>;
 		status = "okay";
+		/* Generic PHY, providing serdes lanes */
+		phys = <&cp1_comphy5 1>;
 	};
 };
 
 &cp1_pcie0 {
 	status = "okay";
 	msi-parent =<&ap0_gic_its>;
+
+	/* Generic PHY, providing serdes lanes */
+	phys = <&cp1_comphy0 0
+		&cp1_comphy1 0
+		&cp1_comphy2 0
+		&cp1_comphy3 0>;
+	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp1_pcie1 {
-- 
2.17.1

